src/terminal/adapter/DispatchTypes.hpp

```diff
@@ -216,7 +216,7 @@ namespace Microsoft::Console::VirtualTerminal::DispatchTypes
 
     enum class CursorStyle : unsigned int
     {
-        BlinkingBlock = 0,
+        BlinkingBlock = 0, // Implemented as "restore cursor to user default".
         BlinkingBlockDefault = 1,

```

Comment:
Instead of adding a comment here, couldn't we just rename the enum value to something like `UserDefault`, and then also rename `BlinkingBlockDefault` to `BlinkingBlock`. That seems like it would be a better indication of what these values actually represent.

Reply:
FYI, when you've resolved an issue that I've raised, and you want to tidy up the comments, you can mark the thread as resolved yourself if you want to. I don't actually have the ability to resolve issues on someone else's PR - I think that's something only MS devs can do.

Reply:
Yes. Thanks @j4james. I keep forgetting this "resolving comments" feature. I think I got the ability to do this after Dustin granted me the authority, not because I'm a MS dev. BTW I work for Azure in MS, not Console team. So strictly speaking I'm still a community member 😉 .